deploy-board/deploy_board/webapp/error_views.py

```diff
@@ -40,7 +40,12 @@ def process_exception(self, request, exception):
         # Error is displayed as a fragment over related feature area
         if request.is_ajax():
             ajax_vars = {'success': False, 'error': str(exception)}
-            return HttpResponse(json.dumps(ajax_vars), content_type='application/javascript')
+            ret = 500
+            if isinstance(exception, NotAuthorizedException):
+                ret = 403
+            elif isinstance(exception, FailedAuthenticationException):
+                ret = 401
+            return HttpResponse(json.dumps(ajax_vars), status=ret, content_type='application/javascript')
```

Comment:
It'd be nice if we can handle more exception types defined in deploy-board/deploy_board/webapp/helpers/exceptions.py

Reply:
there are only 3 exceptions in this file. So, I can add another 404 case.